shared/src/main/scala/pl/metastack/metatime/PatternTime.scala

```diff
@@ -0,0 +1,66 @@
+package pl.metastack.metatime
+
+case class PatternTime(parts: Seq[Time => String] = Nil) {
+  def string(s: String): PatternTime = add(_ => s)
+
+  def dash : PatternTime = string("-")
```

Comment:
Actually I tried to create Pattern to have common functions and i put following functions there:

  /*def dash : PatternDateTime = string("-")
  def space: PatternDateTime = string(" ")
  def colon: PatternDateTime = string(":")
  def comma: PatternDateTime = string(",")
  def dot  : PatternDateTime = string(".")*/

But this requires me to defined string function also so I did

 def string(s: String): PatternDateTime = add(_ => s)

but then it requires me to define add method as well while we have add methods specific for DateTime and Time (separately). What do you suggest?